### PR TITLE
Feedback: backfill script_level_id, part 1

### PR DESCRIPTION
**Pull request by Erin007**
Partial fulfillment of [LP-590](https://codedotorg.atlassian.net/browse/LP-590) 

To retrieve accurate information to display on the All Feedback page, `script_level_id` needs to be stored for each TeacherFeedback. With #29671 when a new TeacherFeedback is created we will begin populating that field. However, we need to backfill `script_level_id` for previously created TeacherFeedbacks. 

The most straightforward and by far most common scenario to backfill is when a `Level` is associated with only one `ScriptLevel`.  Here I wrote a script that will find all of the TeacherFeedbacks where `script_level_id` is nil, then check how many `ScriptLevel`s are associated with the `Level` associated with the feedback's `level_id`. If it is only one `ScriptLevel` then I set the `script_level_id` for the feedback to the correct value. Scenarios in which the level is associated with multiple script levels will be handled next. 

When I ran it locally, the script logged what I expected and updated the `script_level_id` field correctly:
backfill-success

```[#<TeacherFeedback:0x00007f8f465a0488
  id: 1,
  comment: "hey do your work",
  student_id: 214,
  level_id: 11047,
  teacher_id: 1,
  created_at: Wed, 26 Jun 2019 16:50:05 UTC +00:00,
  updated_at: Fri, 12 Jul 2019 20:11:58 UTC +00:00,
  deleted_at: nil,
  performance: nil,
  student_visit_count: 5,
  student_first_visited_at: Fri, 28 Jun 2019 00:35:29 UTC +00:00,
  student_last_visited_at: Wed, 03 Jul 2019 23:33:10 UTC +00:00,
  **script_level_id: 23948,**
  seen_on_feedback_page_at: nil>,
 #<TeacherFeedback:0x00007f8f465a02a8
  id: 2,
  comment: "feedback #2",
  student_id: 214,
  level_id: 11039,
  teacher_id: 1,
  created_at: Wed, 26 Jun 2019 17:06:42 UTC +00:00,
  updated_at: Fri, 12 Jul 2019 20:11:59 UTC +00:00,
  deleted_at: nil,
  performance: nil,
  student_visit_count: nil,
  student_first_visited_at: nil,
  student_last_visited_at: nil,
  **script_level_id: 23946,**
  seen_on_feedback_page_at: nil>,
 #<TeacherFeedback:0x00007f8f465a0168
  id: 3,
  comment: "what's the script_level_id?!?!",
  student_id: 214,
  level_id: 15568,
  teacher_id: 1,
  created_at: Thu, 11 Jul 2019 21:29:25 UTC +00:00,
  updated_at: Fri, 12 Jul 2019 20:12:00 UTC +00:00,
  deleted_at: nil,
  performance: nil,
  student_visit_count: 2,
  student_first_visited_at: Thu, 11 Jul 2019 21:33:00 UTC +00:00,
  student_last_visited_at: Fri, 12 Jul 2019 17:01:29 UTC +00:00,
  **script_level_id: 30836,**
  seen_on_feedback_page_at: nil>,
 #<TeacherFeedback:0x00007f8f465a0028
  id: 4,
  comment: "did this save the script_level_id?",
  student_id: 214,
  level_id: 15568,
  teacher_id: 1,
  created_at: Fri, 12 Jul 2019 18:01:49 UTC +00:00,
  updated_at: Fri, 12 Jul 2019 20:12:00 UTC +00:00,
  deleted_at: nil,
  performance: nil,
  student_visit_count: nil,
  student_first_visited_at: nil,
  student_last_visited_at: nil,
  **script_level_id: 30836,**
  seen_on_feedback_page_at: nil>,
 #<TeacherFeedback:0x00007f8f4659be88
  id: 5,
  comment: "did this save the script_level_id? how about now?",
  student_id: 214,
  level_id: 15568,
  teacher_id: 1,
  created_at: Fri, 12 Jul 2019 18:08:35 UTC +00:00,
  updated_at: Fri, 12 Jul 2019 20:12:00 UTC +00:00,
  deleted_at: nil,
  performance: nil,
  student_visit_count: nil,
  student_first_visited_at: nil,
  student_last_visited_at: nil,
  **script_level_id: 30836,**
  seen_on_feedback_page_at: nil>,
 #<TeacherFeedback:0x00007f8f4659bca8
  id: 6,
  comment:
   "did this save the script_level_id? how about now? aha! what if I add script_level_id to permitted params?",
  student_id: 214,
  level_id: 15568,
  teacher_id: 1,
  created_at: Fri, 12 Jul 2019 18:11:55 UTC +00:00,
  updated_at: Fri, 12 Jul 2019 20:12:00 UTC +00:00,
  deleted_at: nil,
  performance: nil,
  student_visit_count: nil,
  student_first_visited_at: nil,
  student_last_visited_at: nil,
  **script_level_id: 30836,**
  seen_on_feedback_page_at: nil>,
 #<TeacherFeedback:0x00007f8f4659bb68
  id: 7,
  comment: "did this save the script_level_id?!?!",
  student_id: 214,
  level_id: 15568,
  teacher_id: 1,
  created_at: Fri, 12 Jul 2019 19:06:51 UTC +00:00,
  updated_at: Fri, 12 Jul 2019 19:06:51 UTC +00:00,
  deleted_at: nil,
  performance: nil,
  student_visit_count: nil,
  student_first_visited_at: nil,
  student_last_visited_at: nil,
  **script_level_id: 30836,**
  seen_on_feedback_page_at: nil>]

### CODE REVIEW

File: dashboard/scripts/archive/backfill_script_level_ids_for_teacher_feedbacks_1.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+end
+
+def puts_count
+  puts "There are #{feedbacks_without_script_level_id.count} feedbacks without a script_level_id"
+end
+
+def update_script_level_ids
+  puts "backfilling script_level_ids..."
+  feedbacks_without_script_level_id.find_each do |feedback|
+    associated_script_levels = Level.find(feedback.level_id).script_levels
```
Comment (islemaster) [inline dashboard/scripts/archive/backfill_script_level_ids_for_teacher_feedbacks_1.rb]: I think you can just:
```suggestion
    associated_script_levels = feedback.level.script_levels
```

**Review by islemaster — APPROVED** [review]
This looks great!

**Review by maddiedierker — APPROVED** [review]
🥇

**Comment by Erin007** [comment]
I ran the script on production: 
Screen Shot 2019-07-16 at 12 41 13 PM

**Pull request merged**